src/qt/askpassphrasedialog.cpp

```diff
@@ -234,6 +237,15 @@ bool AskPassphraseDialog::event(QEvent *event)
     return QWidget::event(event);
 }
 
+void AskPassphraseDialog::toggleShowPassword(bool show)
+{
+    ui->toggleShowPasswordButton->setDown(show);
+    auto mode = show ? QLineEdit::Normal : QLineEdit::Password;
```

Comment:
nit: does the `auto` here make sense? IMO this is a way of using auto that leads to confusion reading the code.

Reply:
`auto` is one of the rare items in C++11 I go back and forth on.  My general heuristic is that if the thing declared as `auto` is used in at most the next five lines, and the type of the variable doesn't add anything meaningful to the understanding of the code, then auto is fine (and maybe slightly preferable).

In this case, `mode` is used just in the following three lines as an argument to a descriptively named setter.  I don't think explicitly declaring it as `QSomeEnumType` adds anything here.

Reply:
IMO `const auto mode = ...` and the RHS is descriptive enough.